primefaces/src/main/java/org/primefaces/virusscan/impl/ClamDaemonScanner.java

```diff
@@ -57,8 +57,7 @@ public class ClamDaemonScanner implements VirusScanner {
 
     @Override
     public boolean isEnabled() {
-        ExternalContext ctx = FacesContext.getCurrentInstance().getExternalContext();
-        return ctx.getInitParameter(CONTEXT_PARAM_HOST) != null && ctx.getInitParameter(CONTEXT_PARAM_PORT) != null;
+        return true;
```

Comment:
I have to take a deeper dive into where this enabled flag is used but does this turn on ClamAv automatically when it shouldn't?

Reply:
the virusscan is controlled by the corresponding virusScan attribute of the fileUpload component, isn't it?

Reply:
But what happens if VirusTotal and ClamAV both return true here it will automatically try to use this one even if a user doesn't want to right?

Reply:
You have that correct.  But its pluggable with 3 different virus scanning engines either VirusTotal, ClamAV, or a Custom one from the user.  This flag controls whether to "try" this virusanner or not.  Previously it would only be attempted if the user had configued ClamAV. Now you are forcing users using `viruscan="true"` to ALWAYS use ClamAV when they don't want to.

Reply:
ok. then it would be needed a possibility to switch-off the default ClamAV scanner.

what about introducing

  <context-param>
    <param-name>primefaces.virusscan.CLAMAV_DEFAULT</param-name>
    <param-value>false</param-value>
  </context-param>

wdyt?

Reply:
But my original solution already worked.  You have to defined a server and port ANYWAY and that is what triggered the enablement.    So now you are introducing a new parameter for no reason all just to not have someone fill out server and port?

Reply:
+1 melloware